tests/integration/kubernetes/k8s-confidential-attestation.bats

```diff
@@ -72,6 +72,36 @@ setup() {
 	kubectl logs aa-test-cc | grep -q "aatest"
 }
 
+@test "Negative policy test" {
```

Comment:
Nit: I suggest a new test name "Cannot get CDH resource when deny-all policy is set"